libbeat/processors/add_docker_metadata/add_docker_metadata.go

```diff
@@ -132,6 +132,8 @@ func lazyCgroupCacheInit(d *addDockerMetadata) {
 	}
 }
 
+// Run runs the processor that adds container-related fields to the event
+// This processor does not modify any `Meta` fields in the event, it accesses and modifies `event.Fields` directly
```

Comment:
Why did you change it? Do we win anything by touching processors that are not impacted?

Reply:
yes, now it looks more obvious that the processor is not allowed to edit metadata and works with the `Fields` only. I still think it's a good idea.